### ISSUE TITLE
Automatic dependencies generation

### ISSUE THREAD

**Issue by mkyc** [epiphany-platform/m-aws-kubernetes-service#9]
Extension of #5. We should have it automated to generate it into documentation.

**Comment by rafzei** [comment]
@mkyc do we have a similar solution implemented in go-lang somewhere? (Im not sure but...  was it done by you somewhere in e-wrapper?)

**Comment by mkyc** [comment]
Yeah, epiphany cli is having task to export all used packages in [here](https://github.com/epiphany-platform/cli/blob/develop/Makefile#L49-L54) but I don't think it's exactly what I meant. I wanted to have something what would generate [Module dependencies](https://github.com/epiphany-platform/m-aws-kubernetes-service#module-dependencies) section in readme file or maybe some other file linked from there. This task is probably bit of research how to prepare script or generally where to store versions of everything used in module and how to generate documentation automatically with some `make documentation` target.
